packages/client/src/views/RegisterForm/ApplicationForm.tsx

```diff
@@ -70,14 +63,6 @@ function mapStatetoProps(
   }
 }
 
-export const ApplicationForm = connect<
-  StateProps,
-  {},
-  IFormProps &
```

Comment:
The `ownProps` for `ApplicationForm` is only `RouteComponentProps` as this Component is rendered directly from `<Route />` so IFormProps should not be here.